daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/v2/ledgerinteraction/IdeLedgerClient.scala

```diff
@@ -283,6 +283,90 @@ class IdeLedgerClient(
       }
   }
 
+  private def getTypeIdentifier(t: Ast.Type): Option[Identifier] =
+    t match {
+      case Ast.TTyCon(ty) => Some(ty)
+      case _ => None
+    }
+
+  private def fromInterpretationError(err: interpretation.Error): SubmitError = {
+    import interpretation.Error._
+    err match {
+      case e: RejectedAuthorityRequest => SubmitError.UnknownError(e.toString)
```

Comment:
Might want to leave comments drawing attention to this, so that the next person making changes knows not to leak info.